plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcRecordCursor.java

```diff
@@ -63,7 +64,7 @@
     private ResultSet resultSet;
     private boolean closed;
 
-    public JdbcRecordCursor(JdbcClient jdbcClient, ExecutorService executor, ConnectorSession session, JdbcSplit split, JdbcTableHandle table, List<JdbcColumnHandle> columnHandles)
+    public JdbcRecordCursor(JdbcClient jdbcClient, ExecutorService executor, ConnectorSession session, JdbcSplit split, ConnectorTableHandle table, List<JdbcColumnHandle> columnHandles)
```

Comment:
Let's have a `BaseJdbcConnectorTableHandle` marker interface and let JdbcProcedureHandle, JdbcTableHandle both extend it; use here

Reply:
It should be as an abstract class right ?